static/js/theme-map/Maps/ProviderMap.js

```diff
@@ -24,6 +24,15 @@ class ProviderMapOptions {
     this.zoomEndHandler = () => {};
     this.canvasClickHandler = () => {};
     this.providerOptions = {};
+    this.language = 'en';
+  }
+
+  /**
+   * @param {string} language The language displayed on map
+   */
+  withLanguage(language) {
```

Comment:
For some reason I thought the full page map already knew about your locale to some degree. Could we just use the locale to determine your language?

Reply:
MapBoxMap class doesn't have access to locale or language from what I can see. I console log out its content and params given from constructor and couldn't find anything either. So I had to pass the language it. User wouldn't need to add anything in their configuration, `ThemeMap` would take the user provided locale and pass that accordingly to the actual Map provider instance.